wait_for_port

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+port="${1:-80}"
+timeout="${2:-60}"
+host="${3:-localhost}"
+
+echo "Waiting for someone listening on port $port"
+timeout "$timeout" sh -c 'while ! nc -q0 -w1 -z '"$host"' '"$port"' </dev/null >/dev/null 2>&1; do sleep 1; done'
```

Comment:
Probably can be in the base image?

Reply:
It should definitely be (also took a look at other tools), but the base image doesn't exist yet.

Reply:
(the `COPY` step is in the "move into base image" section)